cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -9,8 +9,18 @@ import (
 
 // Supervisor is the Supervisor config file format.
 type Supervisor struct {
-	Server *OpAMPServer
-	Agent  *Agent
+	Server       *OpAMPServer
+	Agent        *Agent
+	Capabilities *Capabilities `mapstructure:"capabilities"`
+}
+
+// Capabilities is the set of capabilities that the Supervisor supports.
+type Capabilities struct {
+	AcceptsRemoteConfig    *bool `mapstructure:"accepts_remote_config"`
+	ReportsEffectiveConfig *bool `mapstructure:"reports_effective_config"`
+	ReportsOwnMetrics      *bool `mapstructure:"reports_own_metrics"`
+	ReportsHealth          *bool `mapstructure:"reports_health"`
+	ReportsRemoteConfig    *bool `mapstructure:"reports_remote_config"`
```

Comment:
From what I understand the default value for all this flags is `false`, right? If that's the case do they need to be pointers to `bool` or simply `bool` is enough?

Reply:
The [specification](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/opampsupervisor/specification#supervisor-configuration) says that these should be enabled by default, so I think `*bool` is probably the best approach here.